core/commands/swarm.go

```diff
@@ -213,6 +232,18 @@ var swarmPeeringRmCmd = &cmds.Command{
 				return err
 			}
 		}
+
+		save, _ := req.Options[swarmSaveOptionName].(bool)
+		if save {
+			update := func(cfg *config.Config) {
+				cfg.Peering.Peers = node.Peering.ListPeers()
```

Comment:
Same as above (but it should filter instead of append)

Reply:
fixed here [35efa59](https://github.com/ipfs/kubo/pull/9425/commits/35efa59ee656069a08f2aef466c922d3847410c0)